### PR TITLE
Feature/java doc

### PR DESCRIPTION
**Pull request by laSinteZ**
### Description of the Change
Added "Java library" section. Kudos to @muratovv  for script and explanations :)
[Here is the compiled page](https://html-xrwpptoamt.now.sh/guides/libraries.html#java-library) to simplify the review process

### Benefits
Now we have an explanation of how to add the Java Client library to the Java project.

### Possible Drawbacks
None

<!-- If no drawbacks, explicitly mention this (write Non

### CODE REVIEW

**Comment by l4l** [comment]
Rebase plz

File: docs/source/guides/libraries.rst [hunk 1]
```diff
@@ -19,17 +19,150 @@ Troubleshooting
...
+
+Now we need to build our Java native library. Run 
+``example/java/build_library.sh``, which will take care of it.
+
+.. code-block:: shell
+
+  cd example/java
+  ./build_library.sh
+
+.. note:: ``build_library.sh`` script creates a ``dist`` folder with
```
Comment (neewy) [inline docs/source/guides/libraries.rst]: why dist by the way?

**Comment by neewy** [comment]
There is one thing missing:  in `example/java/TransactionExample.java` we don't have code which explains how to get status of a transaction using gRPC streaming. @muratovv should this example be added in a separate PR?

File: docs/source/guides/libraries.rst (hunk 1 above)
Comment (muratovv) [reply docs/source/guides/libraries.rst]: Short form of `distribution`. What is more suitable for it? @neewy

**Pull request merged**